addons/website_mail_channel/__init__.py

```diff
@@ -1,4 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Note for commit 4 ([REM] website_mail_channel: remove the module "website_mail_channel"): also add a rationale and explain a bit more what you are doing :) otherwise I think that people outside of mail scope do not really understand all those steps.

Note that all those are cleaning steps that will make your PR merge message, meaning you will work once and use it twice :D ! #motivational #writingstuff